lms/templates/theme-variables.html

```diff
@@ -122,13 +122,13 @@
       },
       {
         'link_title': 'Privacy Policy',
-        'link_URL': '/privacy',
+        'link_URL': 'https://pass.texasgateway.org/tos',
```

Comment:
I just want to do a sanity check here. `Privacy Policy` and `Terms of Services` are both pointing to the same page. I checked out that page and it does in fact have ToS and privacy info. Are you sure Trinity still wants two links here? Is there any chance they want to get rid of one or combine them into a single link that says `Terms of Service and Privacy Policy`?